server/src/main/java/keywhiz/service/daos/SecretSeriesMapper.java

```diff
@@ -20,26 +20,36 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import java.io.IOException;
 import java.util.Map;
+import java.util.Optional;
 import javax.inject.Inject;
 import keywhiz.api.ApiDate;
+import keywhiz.api.model.Group;
 import keywhiz.api.model.SecretSeries;
 import keywhiz.jooq.tables.records.SecretsRecord;
+import keywhiz.service.config.Readonly;
 import org.jooq.RecordMapper;
 
 public class SecretSeriesMapper implements RecordMapper<SecretsRecord, SecretSeries> {
   private static final TypeReference<Map<String, String>> MAP_STRING_STRING_TYPE =
       new TypeReference<>() {};
+
   private final ObjectMapper mapper;
+  private final GroupDAO groupDAO;
 
-  @Inject public SecretSeriesMapper(ObjectMapper mapper) {
+  @Inject public SecretSeriesMapper(
+      ObjectMapper mapper,
+      @Readonly GroupDAO groupDAO) {
     this.mapper = mapper;
+    this.groupDAO = groupDAO;
   }
 
   public SecretSeries map(SecretsRecord r) {
+    String owner = getOwner(r);
+
     return SecretSeries.of(
         r.getId(),
         r.getName(),
-        null,
+        owner,
         r.getDescription(),
         new ApiDate(r.getCreatedat()),
         r.getCreatedby(),
```

Comment:
Concretely, this is the major behavior change on the read path: if the database contains a value for the owner it will now be represented in our API model.